client/my-sites/stats/stats-list/stats-list-card.jsx

```diff
@@ -29,15 +31,20 @@ const StatsListCard = ( {
 		if ( listItemData?.page ) {
 			gaRecordEvent( 'Stats', ` Clicked ${ moduleNameTitle } Summary Link in List` );
 			page( listItemData.page );
-		} else {
-			// downloads component
+		} else if ( listItemData?.link ) {
+			// downloads component and some old search items (not all)
 			gaRecordEvent( 'Stats', ` Clicked ${ moduleNameTitle } External Link in List` );
 
 			window.open( listItemData?.link );
 		}
 	};
 
-	const barMaxValue = data?.[ 0 ]?.value || 0;
+	let barMaxValue = 0;
+
+	// Search doesn't have items sorted by value when there are 'Unknown search terms' present.
+	data?.forEach( ( item ) => {
+		barMaxValue = item?.value > barMaxValue ? item?.value : barMaxValue;
+	} );
```

Comment:
```suggestion
    // Search doesn't have items sorted by value when there are 'Unknown search terms' present.
	const barMaxValue = Math.max( ...data?.map( item => item.value ).filter( Number.isFinite ) );
```

A little code golf for fun 🙂 
(Warning: Hasn't been checked for edge cases!)

Reply:
Oh, I like it! Thanks, I'll merge and test.